demo/src/routes/menu/CollapsibleSection.svelte

```diff
@@ -0,0 +1,50 @@
+<script>
+	import {collapseVerticalTransition} from '@agnos-ui/core/services/transitions/bootstrap';
+	import {createTransition} from '@agnos-ui/core/services/transitions/baseTransitions';
+	import {writable} from '@amadeus-it-group/tansu';
+
+	export let headerText;
+	const paramTransition$ = writable(collapseVerticalTransition);
+	const paramAnimation$ = writable(true);
+	const paramVisible$ = writable(false);
```

Comment:
shouldn't be open by default?

Reply:
we decided to open only the currently visible section by default (e.g. Getting started, when the Introduction is open), so that the space is preserved